shared/actions/chat2/index.js

```diff
@@ -742,6 +742,27 @@ const setupEngineListeners = () => {
       `ChatHandler: got NotifyChat.ChatSetConvSettings with no valid minWriterRole for convID ${conversationIDKey}. The local version may be out of date.`
     )
   })
+  engine().setIncomingActionCreators(
+    'chat.1.NotifyChat.ChatPaymentInfo',
+    (notif: RPCChatTypes.NotifyChatChatPaymentInfoRpcParam) => {
+      const conversationIDKey = Types.conversationIDToKey(notif.convID)
+      const paymentInfo = Constants.uiPaymentInfoToChatPaymentInfo(notif.info)
+      if (!paymentInfo) {
+        // This should never happen
```

Comment:
If it shouldn't ever happen, perhaps a black bar in the case that it does? Otherwise we just won't notice the logger.error()

Reply:
Sounds good